TaskHero/ParseClient.swift

```diff
@@ -83,6 +82,49 @@ class ParseClient: NSObject {
         }
     }
     
+    func createTaskInstance(task: Task, success: @escaping () -> (), failure: @escaping (Error) -> ()) {
+        let u = PFUser()
+        u.objectId = User.current!.id
+        
+        let t = PFObject(className: "TaskInstances")
+        t["author"] = u
+        t["name"] = task.name
+        t["details"] = task.details
+        t["estimated_time"] = task.estimatedTime
+        t["chat_id"] = "test_chat_id"
+        t["task_id"] = task.taskPFObject
```

Comment:
The task_id thing is throwing me off. Is this field only going to be populated if the task is a task instance? We didn't give the task a regular id field